.github/workflows/bionic_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Bionic-Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  industrial_ci:
+    name: Melodic
+    runs-on: ubuntu-latest
+    env:
+      CI_NAME: Bionic-Build
+      OS_NAME: ubuntu
+      OS_CODE_NAME: bionic
+      ROS_DISTRO: melodic
+      ROS_REPO: main
+      BEFORE_BUILD_TARGET_WORKSPACE: './ci/.ci_install_phoxi_sdk_bionic.sh'
+      BEFORE_BUILD_TARGET_WORKSPACE_EMBED: 'export PHOXI_CONTROL_PATH=/opt/PhotoneoPhoXiControl-'
```

Comment:
Are you not missing the version in the PhoXiControl path after the `-`?

Reply:
No, I don't think so. Whenever I run the install script (`PhotoneoPhoXiControlInstaller-1.2.14-Ubuntu18-STABLE.run`) on Bionic and Focal, it installs the SDK at `/opt/PhotoneoPhoXiControl-`. On Ubuntu Xenial it actually installs at `/opt/PhotoneoPhoXiControl-1.2.14`. I haven't really looked into the discrepancy there, but the build works correctly per the CI result linked above (and on the local machines where I've installed this before)

Reply:
True that. As a side not, you can specify an explicit path where the package should be installed by supplying the path as an installer argument, e.g.: ` ./PhotoneoPhoXiControlInstaller-1.2.14-Ubuntu18-STABLE.run /opt/pxc/1.2.14/` -> this would install PXC to `/opt/pxc/1.2.14`.